OSPSuite.FuncParser.v3.ncrunchsolution

```diff
@@ -0,0 +1,6 @@
+﻿<SolutionConfiguration>
+  <Settings>
+    <AllowParallelTestExecution>True</AllowParallelTestExecution>
+    <SolutionConfigured>True</SolutionConfigured>
+  </Settings>
+</SolutionConfiguration>
```

Comment:
what is this?

Reply:
Ncrunch file to run the tests automatically. Once per solution. Only for C# test projects.